### DISCUSSION TITLE
환불 요청

### DISCUSSION THREAD

**Discussion by chae05** [community/community#60562] [New to GitHub] [answered]
정기 결제가 청구 되었습니다. 사용하지 않을 예정이라 미사용금액에 대해 환불 요청 드립니다.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @chae05 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
